lte/gateway/c/core/oai/tasks/mme_app/mme_app_context.cpp

```diff
@@ -851,14 +841,12 @@ void mme_ue_context_update_ue_sig_connection_state(
     OAILOG_FUNC_OUT(LOG_MME_APP);
   }
   if (ue_context_p->ecm_state == ECM_CONNECTED && new_ecm_state == ECM_IDLE) {
-    hash_rc = hashtable_uint64_ts_remove(
-        mme_ue_context_p->enb_ue_s1ap_id_ue_context_htbl,
-        (const hash_key_t)ue_context_p->enb_s1ap_id_key);
-    if (HASH_TABLE_OK != hash_rc) {
+    if ((mme_ue_context_p->enb_ue_s1ap_key2mme_ueid_map.remove(
+            ue_context_p->enb_s1ap_id_key)) != magma::PROTO_MAP_OK) {
       OAILOG_WARNING_UE(LOG_MME_APP, ue_context_p->emm_context._imsi64,
-                        "UE context enb_ue_s1ap_ue_id_key %ld "
+                        "Failed to remove enb_ue_s1ap_ue_id_key %u "
```

Comment:
Correct the format specifier for "enb_ue_s1ap_ue_id_key".Some times it's used "%u" and at other times it's used as "%ld"